foundry.toml

```diff
@@ -23,7 +23,7 @@ build_info = true
 extra_output = ["storageLayout"]
 
 [profile.ci]
-fuzz = { runs = 10_000 }
+fuzz = { runs = 16 }
```

Comment:
should this be for all CI runs? Ideally I'd run gas report on say 16 runs then run the rest of the tests on the original number to ensure we catch issues